config/samples/cluster_v1alpha1_eks.yaml

```diff
@@ -26,12 +30,3 @@ spec:
               - effect: "Allow"
                 action: ["*"]
                 resource: ["*"]
-    - name: "g2"
```

Comment:
May be not a bad idea to keep this in example to show it can create multiple nodegroups.

Reply:
Of course, have included the same. Thanks for catching this